data/core/units/dwarves/Explorer.cfg

```diff
@@ -34,11 +34,12 @@
     [/attack]
     [attack]
         name=axe
-        description= _"axe"
+        description= _"battle axe"
```

Comment:
I find it very odd that a dwarf would use an expensively crafted weapon such as a battle axe as a throwing weapon. Rather it should be kept as "axe", or renamed to "hatchet" like the one the Dwarf lord uses.

Reply:
Ok, makes sense. I've reverted this to axe.